### ISSUE TITLE
zpm 0.7.0 unable to load zpm 0.9.0 (from v1-jan2024-updates)

### ISSUE THREAD

**Issue by isc-tleavitt** [intersystems/ipm#449]
In zpm 0.7.0 if you try to zpm "load" from the v1-jan2024-updates branch it'll balk at the %IPM folder not existing.

This boils down to trying the following sequence of folder names:
```
      Set tListNames = ""
      Set tListNames = tListNames _ $ListBuild(tName)
      Set tListNames = tListNames _ $ListBuild($Translate(tName, "%", "_"))
      Set tListNames = tListNames _ $ListBuild($Translate(tName, "_", ""))
```

But failing because the %IPM folder doesn't exist - we just strip the %. The original design intent for such cases - where the folder name maps differently - was to use FilenameTranslateIdentifier/FilenameTranslateAssociator, which were named unwisely. (Every time I go to do so I either need to look at the %ZPM source code or the documentation for $translate.) The primary relevance here is for package manager-aware source control extensions that need to place the file in the right place on the filesystem; we really need a one-to-one mapping rather than saying anything goes. That said, we could (likely needed for 0.9.0 / 1.0.0) store what's actually discovered during the Reload process in the persistent Resource record itself, then refer back to that in the resource processor's OnItemRelativePath.

Ideally 0.7.0 would respect this in Reload and find the directory to load based on it:
`<Resource Name="%IPM.PKG" FilenameTranslateIdentifier=".%" FilenameTranslateAssociator="/" />`

But we also have some changes to the default behavior re: FilenameTranslateIdentifier/FilenameTranslateAssociator - 0.7.0 doesn't strip % by default, while 0.9.0 (and the HS package manager before it) did. To account for this, we need to keep the wacky "look in all the folders" approach.

Bottom line: we need to another folder to the list we look in:
```
      Set tListNames = tListNames _ $ListBuild($Translate(tName, "%", ""))
```

And we're going to need a 0.7.1 release prior to 0.9.0 to fix this.
